eks/modules/autoscaler/variables.tf

```diff
@@ -0,0 +1,38 @@
+variable "name" {
+  description = "Prefix for resource names and tags"
+  type        = string
+}
+
+variable "region" {
+  description = "Region for AWS resources"
+  type        = string
+}
+
+# The cluster autoscaler major and minor versions must match your cluster.
+# For example if you are running a 1.16 EKS cluster set version to v1.16.5
+# See https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/docs/autoscaling.md#notes
+variable "autoscaler_version" {
+  description = "EKS autoscaler image tag"
+  type        = string
+}
+
+variable "autoscaler_chart_version" {
+  description = "EKS chart version"
+  type        = string
+}
+
+variable "openid_connect_url" {
+  description = "OpenId connect provider url"
+  type        = string
+}
```

Comment:
Maybe it will be good to add some validations?

Reply:
Variables inside nested modules are reusing variables from root .tftvars so validation here is unnecessary.